memcached-multi-vm-ubuntu/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Multiple Ubuntu 14.04 VMs with memcached service and one VM with Apache and PHP test page",
```

Comment:
Suggestion: A better title when this template gets displayed in azure.com might be something along the lines of "Memcached service cluster using multiple Ubuntu VMs" which puts the focus on the application that this template installs.